deployment/cdk/opensearch-service-migration/cdk.context.json

```diff
@@ -1,17 +1,16 @@
 {
-  "demo-deploy": {
-    "stage": "dev",
-    "engineVersion": "OS_2.9",
-    "domainName": "demo-opensearch-cluster",
-    "dataNodeCount": 2,
-    "openAccessPolicyEnabled": true,
-    "domainRemovalPolicy": "DESTROY",
-    "artifactBucketRemovalPolicy": "DESTROY",
-    "enableDemoAdmin": true,
-    "trafficReplayerServiceEnabled": true,
-    "trafficReplayerEnableClusterFGACAuth": true,
-    "captureProxyESServiceEnabled": true,
+  "default": {
+    "stage": "<STAGE>",
+    "targetCluster": {
+      "endpoint": "<TARGET_CLUSTER_ENDPOINT>",
+      "auth": { "type": "none" }
+    },
+    "sourceCluster": {
+      "endpoint": "<SOURCE_CLUSTER_ENDPOINT>",
+      "auth": { "type": "none" }
```

Comment:
What about having options in here, even if not valid json? like

```
      "auth": { 
         "type": "none|basic|sigv4"
          // if basic
          "username"
          "password_from_secret_arn"
       }
    "vpcId": "<VPC_ID>",
```

Reply:
I like this! Incorporating it now